salt/utils/templates.py

```diff
@@ -54,6 +54,15 @@ def render_tmpl(tmplsrc,
         if context is None:
             context = {}
 
+        # Alias cmd.run to cmd.shell to make python_shell=True the default for
+        # templated calls
+        if 'salt' in kws:
+            if 'cmd.run' in kws['salt'] and 'cmd.shell' in kws['salt']:
+                kws['salt']['cmd.run'] = kws['salt']['cmd.shell']
+            if 'run' in kws['salt'].get('cmd', {}) \
+                    and 'shell' in kws['salt'].get('cmd', {}):
+                kws['salt']['cmd']['run'] = kws['salt']['cmd']['shell']
```

Comment:
No. Note in the previous line, I do `if 'run' in kws['salt'].get('cmd', {})`. If `cmd` isn't present, then the `.get()` will return an empty dict, and the `if 'run' in` will fail.